inlong-dashboard/src/ui/components/CheckCard/index.tsx

```diff
@@ -72,8 +106,7 @@ const CheckCard: React.FC<CheckCardProps> = ({ options, value, onChange, disable
               bodyStyle={{ textAlign: 'center' }}
               className={disabled ? styles.cardDisabled : ''}
             >
-              <DatabaseOutlined style={{ fontSize: 20 }} />
-              <div>{options.find(item => item.value === currentValue)?.label}</div>
+              {renderContent(options.find(item => item.value === currentValue)?.label)}
```

Comment:
Can you add a [tooltip](https://ant.design/components/tooltip)? Some logos have no clear meaning, for example when I hover to `File`.